CSharp/README.md

```diff
@@ -23,6 +24,115 @@ var httpClient = new HttpClient();
 IUserService userService = new UserService();
 ```
 
+### Don't use explicit *this* reference
+
+It is redundant and does not add for readability. For more information read [this SO post](http://stackoverflow.com/questions/5885249/to-use-or-not-to-use-the-this-qualifier-in-c-sharp).
+
+```csharp
+// Bad
+this.ValidateParameters();
+
+// Good
+ValidateParameters();
+```
+
+### Use *string* instead of *String* keyword
+
+```csharp
+// Bad
+String.IsNullOrEmpty(name);
+
+// Good
+string.IsNullOrEmpty(name);
+```
+
+### Use readonly fields when possible
+
+### Use string interpolation

```

Comment:
Should we phrase this as "Prefer string interpolation to string.Format"? Otherwise, people would start creating interpolated strings when there are no parameters to interpolate. (But really, it's kind of an edge case, so not very important.)